sims/vcs/Makefile

```diff
@@ -38,7 +38,7 @@ SIM_FILE_REQS += \
 	$(ROCKETCHIP_RSRCS_DIR)/vsrc/TestDriver.v
 
 # copy files but ignore *.h files in *.f since vcs has +incdir+$(build_dir)
-$(sim_files): $(SIM_FILE_REQS)
+$(sim_files): $(SIM_FILE_REQS) | $(build_dir)
 	mkdir -p $(dir $(sim_files))
```

Comment:
Since you have `| $(build_dir)` in the deps for the rule, I don't think you need this `mkdir` anymore.